debs/siegfried/Makefile

```diff
@@ -0,0 +1,60 @@
+NAME          = siegfried
+PACKAGE       = siegfried
+VERSION       ?= 1.7.3
```

Comment:
AM17 is going to use siegfried 1.7.6. There is already a PR in place: https://github.com/artefactual/archivematica-fpr-admin/pull/65.